src/App.vue

```diff
@@ -102,4 +102,10 @@
   .x-recommendations__item {
     max-width: 142px;
   }
+
+  .x-desktop {
+    .x-empathize {
+      z-index: 1;
```

Comment:
This change is needed because the related tags appear over the empathize during the animation of open and close the empathize.

Reply:
move to here the `width: 100%` of line 98